validator/src/test/java/org/icann/rdapconformance/validator/workflow/rdap/dataset/model/BootstrapDomainNameSpaceTest.java

```diff
@@ -15,7 +15,6 @@ public class BootstrapDomainNameSpaceTest extends BaseUnmarshallingTest<Bootstra
 
     @BeforeMethod
     public void setUp() throws Throwable {
-        //this.bootstrapDomainNameSpace.parse(getClass().getResourceAsStream("/dataset/dns.json"));;
         this.bootstrapDomainNameSpace = deserializer.deserialize(new File(getClass().getResource("/dataset/dns.json").getFile()));
```

Comment:
What is this change for? It seems to be unrelated to a new IANA value.

Reply:
Some cleaning I went through